wordpress-mobile/MediaPicker-iOS
# Move selection indicator and frame in front of overlay view

As part of allowing batch deletion of failed media in WPiOS, we need to allow those failed cells to be selected. At the moment, the overlay view added to failed cells appears on top of the selection indicator and frame, making them less visible. This PR switches the order of things around.

**Before:**

<img width="219" alt="screen shot 2018-02-09 at 16 37 29" src="https://user-images.githubusercontent.com/4780/36038756-768470f8-0db8-11e8-93a0-08db73b7f947.png">

**After:**

<img width="215" alt="screen shot 2018-02-09 at 16 31 51" src="https://user-images.githubusercontent.com/4780/36038765-7a2e88ce-0db8-11e8-9f04-068d2dd06bc8.png">

**To test:**

* Build and run the demo app, and enable the sample overlay cells in options. Then tap the + in the top right corner to bring up the picker. Tap multiple cells to select them, and ensure the selection indicator and frame are visible.
